packages/jest-config/src/normalize.ts

```diff
@@ -1139,6 +1139,13 @@ export default async function normalize(
     newOptions.shard = parseShardPair(argv.shard);
   }
 
+  if (argv.randomize) {
```

Comment:
we should always set `seet` even if `randomize` isn't passed, right? Otherwise the types lie